### PR TITLE
DEVEX-1240: Update VS Code docs about standalone projects

### PR DESCRIPTION
**Pull request by BeFunes**
# Description
VS Code extension docs need to be adjusted to reflect accurately our support for standalone projects.
In this PR, I refer only to "Gitlab" projects, as I'm not sure what language will be used in Docs to refer to "Github projects created through Github app". I'd appreciate help on this! 

# Reasons
DevEx will soon ship support for Standalone project (without re-run with SSH and automatic project detection)
[DEVEX-1240](https://circleci.atlassian.net/browse/DEVEX-1240)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [x] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

[DEVEX-1240]: https://circleci.atlassian.net/browse/DEVEX-1240?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DEVEX-1174]: https://circleci.atlassian.net/browse/DEVEX-1174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: jekyll/_cci2/get-started-with-the-vs-code-extension.adoc [hunk 1]
```diff
@@ -48,12 +48,12 @@ image::{{site.baseurl}}/assets/img/docs/vs_code_extension_login.png[Log in to Ci
 [#set-up-your-project-in-vs-code]
 == 3. Set up your project in VS Code
 
-NOTE: **Using GitLab?** The pipelines manager does not yet support GitLab projects. You can still use the xref:#open-a-config-file-to-test-language-support-features[config helper] for config writing and troubleshooting.
-
 If your VS Code workspace contains one or more CircleCI projects, the extension will detect them automatically, and the pipelines panel will be populated with your most recent pipelines.
 
 If no project is detected, open the extension's settings page (either through the VS Code command `CircleCI: Open Settings`, or by clicking on the settings icon (icon:cog[]) at the top of the pipelines panel), and select your projects manually.
 
+NOTE: **Using GitLab?** The pipelines manager does not yet support automatic detection for GitLab projects. Please select your project manually from the settings page.
```
Comment (kbatuigas) [inline jekyll/_cci2/get-started-with-the-vs-code-extension.adoc:55]:
```suggestion
NOTE: **GitHub App and GitLab projects:** The pipelines manager does not yet support automatic detection for xref:github-apps-integration#[GitHub App] and xref:gitlab-integration#[GitLab] projects. Please select your project manually from the settings page.
```
In the changes we've made to other pages we refer to GitHub App and GitLab integrations, projects and/or users, and then we link to the integration pages which should help GitHub users determine whether they are on GH OAuth or GH App. Something like this should work... I think it's a bit less ambiguous compared to saying "Using GitHub App or GitLab?"

File: jekyll/_cci2/get-started-with-the-vs-code-extension.adoc [hunk 2]
```diff
@@ -92,6 +92,8 @@ Alternatively, you can configure SSH keys for the extension manually through the
 
 To open an SSH session in a dedicated VS Code remote window, you need to install the link:https://marketplace.visualstudio.com/items?itemName=ms-vscode.remote-explorer[official Remote Explorer extension for VS Code].
 
+NOTE: **Using GitLab?** Re-run with SSH from VS Code is not yet enabled for Gitlab projects.
```
Comment (kbatuigas) [inline jekyll/_cci2/get-started-with-the-vs-code-extension.adoc:95]:
```suggestion
NOTE: **GitHub App and GitLab projects:** Re-run with SSH from VS Code is not yet enabled for xref:github-apps-integration#[GitHub App] and xref:gitlab-integration#[GitLab] projects.
```

File: jekyll/_cci2/vs-code-extension-overview.adoc [hunk 3]
```diff
@@ -70,6 +70,8 @@ Expanding a job also lets you:
 [#re-run-with-ssh]
 === Re-run with SSH
 
+NOTE: **Using GitLab?** Re-run with SSH from VS Code is not yet enabled for Gitlab projects.
```
Comment (kbatuigas) [inline jekyll/_cci2/vs-code-extension-overview.adoc:73]:
```suggestion
NOTE: **GitHub App and GitLab projects:** Re-run with SSH from VS Code is not yet enabled for xref:github-apps-integration#[GitHub App] and xref:gitlab-integration#[GitLab] projects.
```

**Review by kbatuigas — CHANGES_REQUESTED** [review]
@BeFunes let me know how these sound!

**Comment by BeFunes** [comment]
@kbatuigas followed all your suggestions, they are all much better than my initial draft. 

On our side the feature is shipped, so as far as I'm concerned this PR can be merged when you're comfortable with it!

**Review by kbatuigas — APPROVED** [review]
Sounds good @BeFunes ! I'll merge this on your behalf, thanks!

**Pull request merged**